cmd/main.go

```diff
@@ -67,6 +69,11 @@ func main() {
 		if len(subset) == 0 {
 			log.Fatalf("Unable to find an element with the class '%s'\n", *byClass)
 		}
+	} else if *byTagName != "" {
+		subset = restify.FindSubnetByTagName(root, atom.Lookup([]byte(*byTagName)))
```

Comment:
It should be named `FindSubsetByTagName` for consistency. README will also need to be tweaked.